account_analytic_tag_distribution/models/account_analytic_tag.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+
+
+class AccountAnalyticTag(models.Model):
+    _name = "account.analytic.tag"
+    _inherit = ["account.analytic.tag", "analytic.mixin"]
```

Comment:
Why the mixin here?

Reply:
The objective (as discussed in https://github.com/OCA/account-analytic/pull/572#issuecomment-1688225050) is to add the analytic distribution in tags, `analytic.mixin` has already defined the fields for it, avoiding adding code for the same purpose. Do you think it should be done with another way?

Reply:
Which fields are you talking about?

Reply:
The `analytic_distribution` field is defined by https://github.com/odoo/odoo/blob/16.0/addons/analytic/models/analytic_mixin.py, this field allows using the analytic_distribution widget to define the analytic distribution on the tag.

Reply:
OK, I see. I was thinking on the traditional one2many field as in v15, but it's OK this way.